jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/handler/ResourceHandlerTest.java

```diff
@@ -664,7 +664,7 @@ public void before() throws Exception
             protected HttpContent.Factory newHttpContentFactory(ByteBufferPool.Sized byteBufferPool)
             {
                 HttpContent.Factory contentFactory = new ResourceHttpContentFactory(getBaseResource(), getMimeTypes(), byteBufferPool);
-                contentFactory = new FileMappingHttpContentFactory(contentFactory);
+                contentFactory = new FileMappingHttpContentFactory(contentFactory, -1, -1);
```

Comment:
This change looks unnecessary as the previously called constructor still exists and also uses -1, -1.

Reply:
Geeeeze - I can never win with @lorban and @sbordet :)   I made this change because of the recent push to get ride of no-args constructors and to make everything explicit.   
I'll remove (as I would have preferred in the first place), but now more confused than ever with other review that I've received!!!